app/services/analyze.py

```
@@ -89,9 +110,22 @@ async def answer_question(
+        decrypted_encryption_salt = self.encryption_service.decrypt(user.encryption_salt)
+        
+        # 4.2) Decrypt the result content
+        for result in results:
+            encrypted_content = result.get("content")
+            
+            if not encrypted_content:
+                continue
+            
+            result['content'] = self._decrypt_content(encrypted_content, decrypted_encryption_salt)
+        
         # 5) README preface: If there’s README content for this repo, stream it first as helpful context.
```

Ruff (0.14.0)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)